simpleflow/execute.py

```diff
@@ -0,0 +1,193 @@
+import subprocess
+import functools
+import inspect
+import json
```

Comment:
As it does not expect large objects to encode, I'd prefer to avoid adding a dependency and a module.